test/cases/api.sh

```diff
@@ -613,6 +665,46 @@ function _instanceWaitSSH() {
   done
 }
 
+function _instanceCheck() {
+  echo "✔️ Instance checking"
+  local _ssh="$1"
+
+  # Check if postgres is installed
+  $_ssh rpm -q postgresql
+
+  pretty_name=$($_ssh grep '^PRETTY_NAME=' /etc/os-release)
+  # Beta image doesn't have available subscription so skip subscription check. See COMPOSER-996
+  if [[ "$ID" =~ "rhel" ]] && [[ ! "$pretty_name" =~ 'Beta' ]]; then
+    # Verify subscribe status. Loop check since the system may not be registered such early
+    set +eu
+    for LOOP_COUNTER in {1..10}; do
+        subscribe_org_id=$($_ssh sudo subscription-manager identity | grep 'org ID')
+        if [[ "$subscribe_org_id" == "org ID: $API_TEST_SUBSCRIPTION_ORG_ID" ]]; then
+            echo "System is subscribed."
+            break
+        else
+            echo "System is not subscribed. Retrying in 30 seconds...($LOOP_COUNTER/10)"
+            sleep 30
+        fi
+    done
+    set -eu
+    [[ "$subscribe_org_id" == "org ID: $API_TEST_SUBSCRIPTION_ORG_ID" ]]
+
+    # Remove all repos except the RHSM repo
+    $_ssh sudo find /etc/yum.repos.d/ ! -name 'redhat.repo' -type f -exec rm -f {} +
+    # Clean cache and get enabled repo list
+    $_ssh sudo rm -rf /var/cache/dnf
+    # Verify package installation. Disable *beta* repos because they are not reachable
+    $_ssh sudo dnf repolist
+    $_ssh sudo dnf -y install dos2unix --disablerepo=*beta*
```

Comment:
hmmm. I'm wondering now that you have the beta SKUs on the account isn't this still going to cause dnf failure if the beta repos are disabled?

Reply:
So I still skip the subscription check in Beta images now...I disable *beta* repos here just in case they blocks the non-beta images. I'm not quite sure if they will appear in GA images.

Reply:
I'm asking RHSM members if they know why the beta repos are unreachable. I still keep the https://issues.redhat.com/browse/COMPOSER-996 open and I'll update there if I get the answer.

Reply:
Asking here in context again. 

If your code is skipping subscriptions on a Beta system (plus that you have a Beta SKU) what's the current problem then ? From all I can tell this section of the script should be skipped so you shouldn't worry about available repos when running on the 8.4 runners?

Reply:
I think you're right. There shouldn't be beta repos in non-beta systems. I'll remove the "--disablerepo=*beta*" part. Thanks!